src/bin/rustfmt.rs

```diff
@@ -189,6 +189,8 @@ fn execute() -> i32 {
             // write_mode is always Plain for Stdin.
             config.write_mode = WriteMode::Plain;
 
+            config.skip_children = true;
```

Comment:
At the moment this is just a wishful thinking. At the moment if you format text from stdin, and it contains `mod foo;`, you get a panic.